ui/js/react/treeherder/revision-list.jsx

```diff
@@ -0,0 +1,95 @@
+'use strict';
```

Comment:
No reason to separate the react files based on the build process - this can be organized any way at all!